src/app/contextBroker/contextBroker.cpp

```diff
@@ -253,6 +253,7 @@ bool            noCache;
 unsigned int    connectionMemory;
```

Comment:
CNR line missing. Maybe something like this:

"Simulated/drop notifications mode is now orthogonal to -notificationMode (Issue XXX)"